### PR TITLE
[Ready] Russian, NT, Synda ammo tech nodes

### PR DESCRIPTION
**Pull request by Trilbyspaceclone**
[Changelogs]
Brakes up Ballistic tech and Illegal Ballistic tech into 3 new nodes!
NT has .45 and all adv WT-550 ammo
Russian has clips, 9mm, n762
Synda has all 10mm and .357
[why]
Russian guns are mega rare, so we should be able to make ammo for them do to them being on the "To few to use"
NT guns shouldn't be illegal and helps brake down the WT-550 into more then on node
Synda guns are uncommon and should be able to get some ammo or such.
Autolathen shouldn't be able to print out speed loaders for such a high damg gun-s

### CODE REVIEW

**Comment by Trilbyspaceclone** [comment]
@kevinz000, I dont understand the error im getting with triv

**Comment by Tupinambis** [comment]
You are going to have to decrease syndie revolver ammo/revolver itself price then, because otherwise it won't ever be worth it.

File: code/modules/research/designs/autolathe_designs.dm [hunk 1]
```diff
@@ -718,14 +718,6 @@
 	build_path = /obj/item/ammo_box/foambox/riot
 	category = list("hacked", "Security")
 
-/datum/design/a357
```
Comment (silicons) [inline code/modules/research/designs/autolathe_designs.dm]: No.

File: code/modules/research/designs/weapon_designs.dm [hunk 2]
```diff
@@ -43,6 +43,30 @@
 	category = list("Ammo")
 	departmental_flags = DEPARTMENTAL_FLAG_SECURITY
 
+/datum/design/a357sl
+	name = "Speed Loader (.357)"
+	id = "a357sl"
+	build_type = PROTOLATHE
+	materials = list(MAT_METAL = 60000)
```
Comment (silicons) [inline code/modules/research/designs/weapon_designs.dm:50]: No

File: code/modules/research/designs/weapon_designs.dm [hunk 3]
```diff
@@ -43,6 +43,30 @@
...
+	build_path = /obj/item/ammo_box/a357
+	departmental_flags = DEPARTMENTAL_FLAG_SECURITY
+
+/datum/design/deagle
+	name = "50ae mag"
+	id = "deagle"
+	build_type = PROTOLATHE
+	materials = list(MAT_METAL = 600000, MAT_TITANIUM = 100000)
```
Comment (silicons) [inline code/modules/research/designs/weapon_designs.dm]: What the fuck

File: code/modules/research/designs/weapon_designs.dm [hunk 4]
```diff
@@ -43,6 +43,30 @@
...
 
+/datum/design/a357sl
+	name = "Speed Loader (.357)"
+	id = "a357sl"
+	build_type = PROTOLATHE
+	materials = list(MAT_METAL = 60000)
+	build_path = /obj/item/ammo_box/a357
+	departmental_flags = DEPARTMENTAL_FLAG_SECURITY
+
+/datum/design/deagle
```
Comment (silicons) [inline code/modules/research/designs/weapon_designs.dm]: Stop

File: code/modules/research/techweb/all_nodes.dm [hunk 5]
```diff
@@ -616,10 +616,37 @@
 	display_name = "Ballistic Weaponry"
 	description = "This isn't research.. This is reverse-engineering!"
 	prereq_ids = list("weaponry")
-	design_ids = list("mag_oldsmg", "mag_oldsmg_ap", "mag_oldsmg_ic")
```
Comment (silicons) [inline code/modules/research/techweb/all_nodes.dm:619]: No.

File: code/modules/research/techweb/all_nodes.dm [hunk 6]
```diff
@@ -616,10 +616,37 @@
...
+/datum/techweb_node/advanced_illegl_ballistics
+	id = "advanced_illegal_ballistics"
+	display_name = "Advanced Illegal Ballistics"
+	description = "Advanced Ballistic for Illegal weaponds."
+	design_ids = list("10mm","10mmap","10mminc","10mmhp", "a357sl")
+	prereq_ids = list("ballistic_weapons","syndicate_basic","explosive_weapons")
+	research_costs = list(TECHWEB_POINT_TYPE_GENERIC = 5000) //Just Synda fireaims
+	export_price = 7000
+
+/datum/techweb_node/oldguns
```
Comment (silicons) [inline code/modules/research/techweb/all_nodes.dm:641]: Just stop.

File: modular_citadel/code/modules/research/techweb/all_nodes.dme.dm [hunk 7]
```diff
@@ -10,10 +10,6 @@
 	. = ..()
 	design_ids += "xenobio_slimeadv"
 
-/datum/techweb_node/ballistic_weapons/New()
-	. = ..()
```
Comment (silicons) [inline modular_citadel/code/modules/research/techweb/all_nodes.dme.dm:14]: No.

**Review by silicons — CHANGES_REQUESTED** [review]
This is horrible horrible balance wise.
Just stop.

**Pull request closed without merging**